requirements_style.txt

```diff
@@ -1,2 +1,3 @@
-codespell==2.1.0
-pydocstyle==6.1.1
+codespell>=2.1.0
+pydocstyle>=6.1.1
```

Comment:
Not sure why these were pinned... freeing them seems to have had no consequences and we should always try to use the latest versions for these sorts of things (because codespell's dictionary is regularly updated)

Reply:
I added the issue https://github.com/pyvista/pyvista/issues/1868 . Thank you for pointing our.

Reply:
> Not sure why these were pinned... freeing them seems to have had no consequences and we should always try to use the latest versions for these sorts of things (because codespell's dictionary is regularly updated)

The reason why I've pinned these is when a new package is released, our PRs have the potential to suddenly fail. This means that new or existing contributors may find their PRs failing for some reason they don't quite understand and pinning the packages helps alleviate this issue.

Reply:
Ah I see. I'd like to keep them unpinned though if possible - the fact that they are in their own GitHub action seperate from the actual tests should make this easy enough to deal with (in theory)

If these fail again ping me and I'll jump on it 😉